pep-0622.rst

```diff
@@ -686,6 +686,16 @@ Finally, all attributes are exposed for matching, if a class wants to hide
 some attributes from matching against them, a custom ``__match__()`` method is
 required.
 
+Exception semantics
+-------------------
+
+While matching each case, the ``match`` statement may trigger execution of other
+functions (for example ``__getitem__()``, ``__len__()`` or ``__get__()`` of some
+descriptor). Almost every exception caused by those propagates outside of the
```

Comment:
Honestly instead of "`__get__` of some descriptor" I'd just use "a property" -- almost everybody knows properties, while their generalization `__get__` feels pretty esoteric.
```suggestion
functions (for example ``__getitem__()``, ``__len__()`` or
a property). Almost every exception caused by those propagates outside of the
```